docs/First-Installation.md

```diff
@@ -9,26 +9,39 @@ This page is for people just getting started on RetroPie. The easiest way to ins
 
-If you hate reading then see this video. Otherwise read on! 
+If you hate reading then see [this video](https://www.youtube.com/watch?v=E1sbnPZ_A8w). Otherwise read on! 
 
 <a href="https://www.youtube.com/watch?v=E1sbnPZ_A8w
-" target="_blank"><img src="https://img.youtube.com/vi/E1sbnPZ_A8w/maxresdefault.jpg" 
-alt="RetroPie First Installation Video" width="1280" height="400" border="10" /></a>
-
-## [Hardware](https://retropie.org.uk/about/building/)
-
- * Raspberry Pi (A, A+, B, B+, 2, Zero, 3, 4) - for best performance use a **Raspberry Pi 4**
- * Raspberry Pi Case (optional but recommended)
- * MicroSD Card (see compatible SD card list [**here**](http://elinux.org/RPi_SD_cards))
- * MicroSD Card Reader (optional - for installing RetroPie if your computer doesn't have a SD card slot)
- * HDMI cable or 4 Pole RCA to 3.5mm Cable (HDMI works best)
- * Television or Computer Monitor- really any screen with HDMI or RCA ports
- * Wifi Dongle or Ethernet Cable (Wifi is built-in for the Pi 3/4 model - see wifi dongle compatible list [**here**](http://elinux.org/RPi_USB_Wi-Fi_Adapters))
- * 5V 2A Micro USB Power Supply (2.5A for Pi 3) / 5.1V 3A USB-C for Pi 4
- * USB Keyboard and Mouse (to get things set up or you can use [SSH](SSH))
- * USB Game Controller of your choice (or you can get the [Control Block](http://blog.petrockblock.com/2014/12/29/controlblock-power-switch-and-io-for-the-raspberry-pi/) to use original SNES controllers)
-
-The simplest way to get most of these components is through a kit such as the [Canakit](https://www.amazon.com/gp/product/B01C6Q2GSY/). 
+" target="_blank"><img src="https://user-images.githubusercontent.com/540857/107464585-9fe22e00-6b2e-11eb-9927-b1ec32360838.jpg" 
+alt="RetroPie First Installation Video" border="10" /></a>
+
+## Hardware
+
+The simplest way to get most of these components is through an all-in-one kit such as the [Canakit](https://www.amazon.com/stores/CanaKit/page/19109D80-639C-40C8-9222-DC6775C304EE).
+
+### Required
+
+ * [Raspberry Pi](https://retropie.org.uk/about/building/)
+ * MicroSD Card ([list of compatible SD cards](http://elinux.org/RPi_SD_cards))
+ * Screen (TV, computer monitor, projector, etc) - anything with HDMI
+ * Video cable (HDMI is best)
+    * Pi 4 will need a [Micro HDMI to HDMI](https://user-images.githubusercontent.com/540857/107463741-c3a47480-6b2c-11eb-8402-7313a490e234.jpg) cable
+    * Pi 1, 2, and 3 will need a full-size [HDMI](https://user-images.githubusercontent.com/540857/107463845-ff3f3e80-6b2c-11eb-8031-3d0ed5563861.jpg) cable
+    * Pi Zero will need a [Mini HDMI to HDMI](https://user-images.githubusercontent.com/540857/107686191-e3de4b80-6c72-11eb-8162-07a82afaac7b.jpg) cable
+    * [4-Pole RCA to 3.5mm](https://user-images.githubusercontent.com/540857/107688722-f3ab5f00-6c75-11eb-9837-ffe7f385b1c5.jpg) is also an option for older screens
+ * Power supply
+    * At least 3.0 amps for Raspberry Pi 4
+    * At least 2.5 amps for Raspberry Pi 3
```

Comment:
Please re-add the voltage requirements, and include the requirements for pi0-2 - see https://www.raspberrypi.org/documentation/faqs/#pi-power-specs